applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/prebuckling_strategy.hpp

```diff
@@ -552,6 +552,12 @@ class PrebucklingStrategy
             // between the current and previous step
             rStiffnessMatrix = rStiffnessMatrixPrevious - rStiffnessMatrix;
 
+            // Symmetrice matrices if enabled
+            if( mMakeMatricesSymmetricFlag ){
+                rStiffnessMatrix = 0.5 * ( rStiffnessMatrix + boost::numeric::ublas::trans(rStiffnessMatrix) );
+                rStiffnessMatrixPrevious = 0.5 * ( rStiffnessMatrixPrevious + boost::numeric::ublas::trans(rStiffnessMatrixPrevious) );
```

Comment:
Please use the SparseSpace for this

Reply:
How does the corresponding function call look like? Shouldn't it be implemented in ublas_space.h then? Because there I can not find a transpose function?

Reply:
Hm ok I thought this was there
Then leave it as is for the moment

Can you please add a comment saying that you are doing it manually bcs the function is missing in the space?